scripts/caclmgrd

```diff
@@ -817,6 +820,46 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
         iptables_cmds.append(self.iptables_cmd_ns_prefix[namespace] + ['ip6tables', '-I', 'INPUT', '2', '-p', 'udp', '-m', 'multiport', '--dports', '3784,4784', '-j', 'ACCEPT'])
         self.run_commands(iptables_cmds)
 
+    def allow_vxlan_port(self, namespace, data):
+        for fv in data:
+            if (fv[0] == "src_ip"):
+                self.VxlanSrcIP = fv[1]
+                break
+
+        if not self.VxlanSrcIP:
+            self.log_info("Received vxlan tunnel configuration without source ip")
+            return False
+
+        iptables_cmds = []
+
+        # Add iptables/ip6tables commands to allow VxLAN packets
+        if ":" in self.VxlanSrcIP:
```

Comment:
@dgsudharsan  Is it better to use this statement to check if it's ipv4 or ipv6? 
`if isinstance(ip_addr, ipaddress.IPv4Address):`

Reply:
Done